content/en/docs/concepts/containers/_index.md

```diff
@@ -13,6 +13,8 @@ card:
 
 <!-- overview -->
 
+Note that "container" is an overloaded term. Whenever you use the word, check whether your audience uses the same definition.
```

Comment:
Maybe
```diff
-Note that "container" is an overloaded term. Whenever you use the word, check whether your audience uses the same definition.
+The word _container_ is an overloaded term. Whenever you use the word, check whether your audience uses the same definition.
```

But! I would put that framing in https://kubernetes.io/docs/contribute/style/style-guide/

In this introductory paragraph, I would an introductory explanation that the page is going to talk about _containers_ and _container images_, and will also explain a little about _containerization_ (using these
technologies in operations and during solution development).

Reply:
Thanks for feedback. I presume you suggested that I add an introductory sentence so I tried to accommodate your feedbacks. 

This is my first PR and I'm not expert in Kubernetes. So, please feel free to edit work. I leave it up to you.